src/pages/audio-rewards-page/config.tsx

```diff
@@ -92,6 +92,24 @@ export const challengeRewardsConfig: Record<
       complete: null
     }
   },
+  // This is used just for the notifications
+  referred: {
+    id: 'referrals' as ChallengeRewardID,
+    title: 'Invite your Friends',
+    icon: <i className='emoji large incoming-envelope' />,
+    description: amount => `Earn ${amount} $AUDIO, for you and your friend`,
+    fullDescription: amount =>
+      `Invite your Friends! You’ll earn ${amount} $AUDIO for each friend who joins with your link (and they’ll get an $AUDIO too)`,
+    progressLabel: '%0/%1 Invites',
+    amount: amounts.referrals,
+    stepCount: 5,
```

Comment:
Referred should be a one-and-done reward, but I'm guessing no impact to the notif of having stepCount = 5?

Reply:
yeah good point. would have nulled all the fields but they're required - i'll fix this anyway to prevent future bugs